android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -274,51 +279,73 @@ private void doMenuAction(int menuItemId) {
         FragmentManager fragmentManager = getSupportFragmentManager();
         switch (menuItemId) {
             case R.id.nav_tracks:
+                addShadowToAppBar(true);
                 fragmentManager.beginTransaction()
                         .replace(R.id.content_frame, new TracksFragment(), FRAGMENT_TAG).commit();
-                getSupportActionBar().setTitle(R.string.menu_tracks);
+                if(getSupportActionBar() != null){
+                    getSupportActionBar().setTitle(R.string.menu_tracks);
+                }
                 break;
             case R.id.nav_schedule:
+                addShadowToAppBar(false);
```

Comment:
@aakashkhatter1296 I believe this should be done after the transaction is complete as if the transaction is not complete, then the shadow won't be there